README.md

```diff
@@ -1,70 +1,206 @@
+### Setup
+
+Clone this repository to your desired folder:
+https://github.com/AbdimulhinYussuf3675/Module2_capstune_Project.git
+
+
+Example commands:
+
+```sh
+  cd my-folder
+  git clone git@github.com:myaccount/Module2_capstune_Project.git
+```
+
+### Install
+
+- Run npm install after cloning the repo
+
+install npx
+
+Example command:
+
+```sh
+  cd my-project
+  npx install
+```
+
+### Usage
+
+- Run npm start for development. 
+- Run npm run build for production
+
+
+### Run tests
+
+To run tests, run the following command:
+- Run npx eslint . for JS check
+- Run npx eslint . --fix to fix JS errors if found
+- Run npx stylelint "**/*.{css,scss}" for CSS check
+- Run npx stylelint "**/*.{css,scss}" --fix to fix CSS errors if found.
+
 
 ### Deployment
 
-This section has moved here: [https://facebook.github.io/create-react-app/docs/deployment](https://facebook.github.io/create-react-app/docs/deployment)
+You can deploy this project using:
+- Github pages
+
+Github pages
+
+<p>follow <a href="https://www.w3schools.com/git/git_remote_pages.asp?remote=github">these steps</a> for example</p>
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+##👥 Authors <a name="authors"></a>
+
+👤**Samuel Tobby**
+
+- GitHub: [@Tobby8629](https://github.com/Tobby8629)
+- Twitter: [@Tobby_samuels](https://twitter.com/Tobby_samuels)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/samuel-popoola-tobby/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🔭 Future Features <a name="future-features"></a>
+- I will add data of states revenue in each country
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🤝 Contributing <a name="contributing"></a>
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](../../issues/).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ⭐️ Show your support <a name="support"></a>
+
+[Ezkiel-udiomuno](https://github.com/Ezekiel-udiomuno) has been of tremendous help so far.
+Give a ⭐️ if you like this project!
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+- I would also appreciate Nelson Sakwa on behance for the access to his [project](https://www.behance.net/gallery/31579789/Ballhead-App-(Free-PSDs)), giving the inspiration to do this project
+- I would like to appreciate [Microverse](https://www.microverse.org/) for providing the resources and the platform for us to be a Professional full-stack developer.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## ❓ FAQ (OPTIONAL) <a name="faq"></a>
+
+- **[Are you sure that you are owner of this work]**
+
+  - [Yes,]
+
+- **[Can you make the work to be responsive]**
+
+  - [yes]
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+
+## 📝 License <a name="license"></a>
 
-### `npm run build` fails to minify
+This project is [MIT](MIT.md) licensed.
```

Comment:
- [optional] your MIT link is broken. please, kindly fix that. we recommend using the [MIT license](https://choosealicense.com/licenses/mit/) - you can set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). You can also use [any other license](https://choosealicense.com/licenses/) if you wish.